c_glib/arrow-glib/compute.cpp

```diff
@@ -3117,6 +3135,7 @@ garrow_sort_key_class_init(GArrowSortKeyClass *klass)
 GArrowSortKey *
 garrow_sort_key_new(const gchar *target,
                     GArrowSortOrder order,
+                    GArrowNullPlacement null_placement,
```

Comment:
Could you add `@null_placement: Whether nulls and NaNs are placed at the start or at the end.` to the docstring?

Reply:
ok